debug/Debug.js

```diff
@@ -0,0 +1,130 @@
+// importing the built version instead of root because dat.gui is badly packaged and exposes ES6
```

Comment:
Debug is neither examples nor src. 

Everything that is in src/ is bundled in the npm package. `debug` won't. 

Examples contains example layers, snippets, demos etc. 

What is in `debug/` is more some internal/debug/dev tools. So it should be in its own folder, called either `debug`, `dev`, `tools` or...